RuntimePermissionsWear/Application/build.gradle

```diff
@@ -20,17 +20,23 @@ plugins {
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
Is this just so it will build right now?

Reply:
Ah, here I'm updating the phone app. So no reason to avoid going to API 31!